README.md

```diff
@@ -20,6 +20,7 @@
   </div>
 </div>
 
+
```

Comment:
We might want to rename the link to "User docs" to something like "MÜSLI integration" or something like that and remove the sentence: "Note the [user docs](https://mampf-hd.github.io/mampf/) are outdated with respect to the screenshots showcased over there."

Reply:
ah good point, fixed in 0ed0c5aa958af6e1